src/main/java/io/vinyldns/java/model/zone/ListZoneChangesResponse.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2018 Comcast Cable Communications Management, LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.vinyldns.java.model.zone;
+
+import java.util.List;
+
+public class ListZoneChangesResponse {
+    private List<ZoneResponse> zoneChanges;
+
+    /**
+     * (optional) The startFrom parameter that was sent in on the HTTP request. Will not be present if
+     * the startFrom parameter was not sent
+     */
+    private String startFrom;
+
+    /**
+     * (optional) The identifier to be passed in as the startFrom parameter to retrieve the next page
+     * of results. If there are no results left, this field will not be present.
+     */
+    private String nextId;
+
+    /** The maxItems parameter that was sent in on the HTTP request. This will be 100 if not sent. */
+    private Integer maxItems;
+
+    /** name filter sent in request */
+    private String nameFilter;
+
+    public ListZoneChangesResponse() {}
+
+    public ListZoneChangesResponse(List<ZoneResponse> zoneChanges,
+                                   String startFrom,
+                                   String nextId,
+                                   Integer maxItems,
+                                   String nameFilter) {
+        this.zoneChanges = zoneChanges;
+        this.startFrom = startFrom;
+        this.nextId = nextId;
+        this.maxItems = maxItems;
+        this.nameFilter = nameFilter;
+    }
+
+    public List<ZoneResponse> getZoneChanges() {
+        return zoneChanges;
+    }
+
+    public void setZoneChanges(List<ZoneResponse> zoneChanges) {
+        this.zoneChanges = zoneChanges;
+    }
+
+    public String getStartFrom() {
+        return startFrom;
+    }
+
+    public void setStartFrom(String startFrom) {
```

Comment:
think logically the responses prob dont need any setters, the attributes can be final